include/openenclave/debugrt/host.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * @file host.h
+ *
+ * This file defines the structures and functions used by the
+ * host-side debugger runtime.
+ */
+#ifndef _OE_DEBUGRT_HOST_H
+#define _OE_DEBUGRT_HOST_H
+
+#include <openenclave/bits/defs.h>
+#include <openenclave/bits/result.h>
+#include <openenclave/bits/types.h>
+#include <openenclave/internal/defs.h> // For OE_STATIC_ASSERT
+
+OE_EXTERNC_BEGIN
+
+#ifdef __linux__
+typedef int32_t wide_char_t;
+#else
```

Comment:
I have simplified this by using wchar.h directly. (Note: not used now; but allows locking it down).
This is host side library and can use system headers. 
Additionally, this library is supposed to be able to exist separately so that any SDK can call into it.